streaming/src/test/scala/org/apache/spark/streaming/DStreamScopeSuite.scala

```diff
@@ -30,28 +30,21 @@ import org.apache.spark.util.ManualClock
 /**
  * Tests whether scope information is passed from DStream operations to RDDs correctly.
  */
-class DStreamScopeSuite extends SparkFunSuite with BeforeAndAfter with BeforeAndAfterAll {
-  private var ssc: StreamingContext = null
-  private val batchDuration: Duration = Seconds(1)
+class DStreamScopeSuite
```

Comment:
This test suites changed initializing SparkContext and StreamingContext "per suite" to "per test" - checking elapsed times...